cmd/influxd/main.go

```diff
@@ -72,9 +72,15 @@ func main() {
 	// Extract name from args.
 	switch cmd {
 	case "run":
```

Comment:
The args are slightly different so still need to check the first is `run` or not.  Using `case "run", ""` causes a panic when starting `influx` w/o any args.